dhis-2/dhis-web/dhis-web-api/src/main/java/org/hisp/dhis/webapi/controller/DataValueController.java

```diff
@@ -506,7 +507,9 @@ public void getDataValueFile(
         @RequestParam( required = false ) String cc,
         @RequestParam( required = false ) String cp,
         @RequestParam String pe,
-        @RequestParam String ou, HttpServletResponse response, HttpServletRequest request )
+        @RequestParam String ou,
+        @RequestParam ( defaultValue = "original" ) String dimension,
```

Comment:
This could be the enum instead of String, then defaultValue would be the enum ORIGINAL